starter-code/iteration-1/main.ts

```diff
@@ -1,18 +1,51 @@
 // Iteration 1
 // 1. Create an array of strings
 
+const tasks: Array<string> = ['This is our first task', 'Eat pizza 🍕 yummy!!!', 'Finish this iteration 1!! 🤓'];
+
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
 
+function addTask(task: string) {
```

Comment:
```function addTask(task: string): number {```

Reply:
remember that you can add types to functions. This one should just return a number.